Fedoraware/TeamFortress2/TeamFortress2/Utils/KeyHelper/KeyHelper.hpp

```diff
@@ -2,43 +2,42 @@
 #include <Windows.h>
 
 class KeyHelper {
```

Comment:
cool

Reply:
i looked into the WndProc method but found it easier to use this method. It seems to work pretty well.